highlighting/generate-rules-highlighting.py

```diff
@@ -84,18 +86,22 @@ def get_translations(KEYWORDS_PATH, KEYWORDS_PATTERN):
 
         yaml_file = yaml.safe_load(keywords_file)
         tmp[language_code] = {}
+        digits[language_code] = []
         for k in yaml_file:
-            if k in TRANSLATE_WORD:
-                tmp[language_code][k] = str(yaml_file[k])
+            if str(k) in TRANSLATE_WORD:
```

Comment:
I am not entirely sure what this does, Do we need to exclude numbers form being translated...?

Can we put it in a more meaningful function or add a comment?

Reply:
The use of digits is different from that of commands.
Digits are used within brackets and that's why I made a special case.

![image](https://user-images.githubusercontent.com/48509466/173021223-5bc3bd2a-8b8b-4274-b50e-6db80235b704.png)
I have added some comments and split into functions